python/lsst/meas/modelfit/display/cModelDisplay.py

```diff
@@ -0,0 +1,264 @@
+    cModelConfig : `lsst.meas.modelfit.CModel(SingleFrame/Forced)Config`
+        Configuration object of the CModel plugin used in the measurement
+        process
+    """
+
+    subImage, devIm, expIm, jointIm = buildCModelImages(exposure, record,
+                                                        cModelConfig)
+    # Get the min and max values for the sub image to use for scaling
+    subImMin = subImage.getArray().min()
+    subImMax = subImage.getArray().max()
+
+    # Calcuate the scaling to use for the residual images
+    devResidual = subImage.getArray()-devIm.getArray()
+    expResidual = subImage.getArray()-expIm.getArray()
+    jointResidual = subImage.getArray()-jointIm.getArray()
+    residualList = (devResidual, expResidual, jointResidual)
+
+    differences = [(x.max()-x.max()) for x in residualList]
+    maxDifferenceIndex = np.argmax(differences)
+    resImMin = residualList[maxDifferenceIndex].min()
+    resImMax = residualList[maxDifferenceIndex].max()
+
+    # Build the image figure
+    fig, axes = plt.subplots(3, 3, sharex='col', sharey='row', figsize=(8, 5))
+    fig.subplots_adjust(left=0.25, right=0.8)
+    lCBar = fig.add_axes([0.1, 0.15, 0.05, 0.7])
+    rCBar = fig.add_axes([0.85, 0.15, 0.05, 0.7])
+
+    # Populate just the exposures in the appropriate places
+    for i in range(3):
+        axes[i, 0].imshow(subImage.getArray(), vmin=subImMin, vmax=subImMax)
+
+    # Populate dev panels
+    axes[0, 1].imshow(devIm.getArray(), vmin=subImMin, vmax=subImMax)
+    axes[0, 2].imshow(devResidual, vmin=resImMin, vmax=resImMax, cmap="BrBG")
+
+    # Populate exp panels
+    axes[1, 1].imshow(expIm.getArray(), vmin=subImMin, vmax=subImMax)
+    axes[1, 2].imshow(expResidual, vmin=resImMin, vmax=resImMax, cmap="BrBG")
+
+    # Populate joint pannels
+    axes[2, 1].imshow(jointIm.getArray(), vmin=subImMin, vmax=subImMax)
+    axes[2, 2].imshow(jointResidual, vmin=resImMin, vmax=resImMax, cmap="BrBG")
+
+    axes[0, 0].set_title("Image")
+    axes[0, 1].set_title("Model")
+    axes[0, 2].set_title('Residuals')
+
+    axes[0, 0].set_ylabel("Dev")
+    axes[1, 0].set_ylabel("Exp")
+    axes[2, 0].set_ylabel("Joint")
+
+    fig.colorbar(axes[0, 0].get_images()[0], lCBar)
+    lCBar.yaxis.set_ticks_position('left')
+    fig.colorbar(axes[maxDifferenceIndex, 2].get_images()[0], rCBar)
+
+    plt.show()
+
+
+def buildCModelImages(exposure, record, cModelConfig):
+    """ Create Images out of the CModel for the given record
+
+    Parameters
+    ----------
+    exposure : `lsst.afw.image.Exposure`
+        Exposure object that contains the source which was  modeled
+    record : `lsst.afw.table.SourceRecord`
+        Record object which contains the measurements made on the source
+    cModelConfig : `lsst.meas.modelfit.CModel(SingleFrame/Forced)Config`
+        Configuration object of the CModel plugin used in the measurement
+        process
+
+    Returns
```

Comment:
It'd be worth considering returning a `pipe.base.Struct` here.  I won't insist on it, but it would avoid the usual problem with remembering the order of returned objects when unpacking.